analyzer/codechecker_analyzer/cmd/checkers.py

```diff
@@ -147,6 +136,13 @@ def add_arguments_to_parser(parser):
                         help="Show details about the checker, such as "
                              "description, if available.")
 
+    parser.add_argument('--label',
+                        nargs='+',
+                        required=False,
+                        default=argparse.SUPPRESS,
+                        help="Filter checkers that are attached the given "
```

Comment:
Can you please add an example section to this and to the check command  when I print the help message of these command similar what we print when running `CodeChecker cmd results -h` (Example scenario:). For example how to get checkers which have severity level of HIGH, or checkers which are in extreme profile etc.

Also in the help message it would be good to mention what kind of profiles do we have and how we can list it. By the way can we list the labels beside the checkers?